services/authenticator.js

```diff
@@ -138,6 +139,14 @@ function Authenticator() {
         return fs.writeFileSync(`${os.homedir()}/.lumberrc`, auth.token);
       });
   };
+
+  this.authenticateAndCreateProject = (config) => {
+    if (config.authToken) {
+      return this.createProject(config);
+    }
+
+    return this.register(config);
```

Comment:
This way it is difficult to understand that the method `register` also create a project. You should either rename the function or do the `createProject` here.